docs/mkdocs-blog.yml

```diff
@@ -49,10 +48,8 @@ plugins:
         default_timezone: "UTC"
         default_time: "06:09"
       enabled: true
-      feed_ttl: 1440
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏can you help me understand why this was removed? Seems like it's unrelated from the other changes.

Reply:
Because this the default value. But I can put it back if you prefer.